diff -upNr Uranium-4.6.1.orig/CMakeLists.txt Uranium-4.6.1.patched/CMakeLists.txt
--- Uranium-4.6.1.orig/CMakeLists.txt	2020-04-21 11:46:12.000000000 +0200
+++ Uranium-4.6.1.patched/CMakeLists.txt	2020-05-09 13:38:28.710327331 +0200
@@ -50,8 +50,9 @@ if(EXISTS /etc/debian_version)
 else()
     install(DIRECTORY UM DESTINATION lib${LIB_SUFFIX}/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR}/site-packages)
 endif()
-install(FILES ${CMAKE_SOURCE_DIR}/cmake/UraniumTranslationTools.cmake
-        DESTINATION ${CMAKE_INSTALL_DATADIR}/cmake-${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}/Modules/ )
+
+install(DIRECTORY cmake DESTINATION ${CMAKE_INSTALL_DATADIR}/uranium)
+install(DIRECTORY scripts DESTINATION ${CMAKE_INSTALL_DATADIR}/uranium)
 install(DIRECTORY resources DESTINATION ${CMAKE_INSTALL_DATADIR}/uranium)
 
 # Detect plugins to install
diff -upNr Uranium-4.6.1.orig/UM/VersionUpgradeManager.py Uranium-4.6.1.patched/UM/VersionUpgradeManager.py
--- Uranium-4.6.1.orig/UM/VersionUpgradeManager.py	2020-04-21 11:46:12.000000000 +0200
+++ Uranium-4.6.1.patched/UM/VersionUpgradeManager.py	2020-05-09 15:02:31.671818993 +0200
@@ -92,21 +92,21 @@ class VersionUpgradeManager:
 
         #Regular expressions of the files that should not be checked, such as log files.
         self._ignored_files = [
-            ".*\.lock",       # Don't upgrade the configuration file lock. It's not persistent.
-            "plugins\.json",  # plugins.json and packages.json need to remain the same for the version upgrade plug-ins.
-            "packages\.json",
-            ".*\.log",        # Don't process the log. It's not needed and it could be really big.
-            ".*\.log.?",      # Don't process the backup of the log. It's not needed and it could be really big.
-            "3.[0-3]\\.*",    # Don't upgrade folders that are back-ups from older version upgrades. Until v3.3 we stored the back-up in the config folder itself.
-            "3.[0-3]/.*",
-            "2.[0-7]\\.*",
-            "2.[0-7]/.*",
-            "cura\\.*",
-            "cura/.*",
-            "plugins\\.*",    # Don't upgrade manually installed plug-ins.
-            "plugins/.*",
-            "./*packages\.json",
-            "./*plugins\.json"
+            r".*\.lock",       # Don't upgrade the configuration file lock. It's not persistent.
+            r"plugins\.json",  # plugins.json and packages.json need to remain the same for the version upgrade plug-ins.
+            r"packages\.json",
+            r".*\.log",        # Don't process the log. It's not needed and it could be really big.
+            r".*\.log.?",      # Don't process the backup of the log. It's not needed and it could be really big.
+            r"3.[0-3]\\.*",    # Don't upgrade folders that are back-ups from older version upgrades. Until v3.3 we stored the back-up in the config folder itself.
+            r"3.[0-3]/.*",
+            r"2.[0-7]\\.*",
+            r"2.[0-7]/.*",
+            r"cura\\.*",
+            r"cura/.*",
+            r"plugins\\.*",    # Don't upgrade manually installed plug-ins.
+            r"plugins/.*",
+            r"./*packages\.json",
+            r"./*plugins\.json"
         ]  # type: List[str]
 
     def registerIgnoredFile(self, file_name: str) -> None:
