diff --git a/CMakeLists.txt b/CMakeLists.txt
index f22d42e..f670fe4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -30,6 +30,8 @@ if(BUILD_PYTHON)
         set(LIB_SUFFIX "")
     endif()
 
+    set(Python3_SITEARCH lib${LIB_SUFFIX}/python3.8/site-packages CACHE STRING "Directory to install Python bindings to")
+
     include_directories(python/ src/ ${SIP_INCLUDE_DIRS} ${Python3_INCLUDE_DIRS})
 endif()
 
